nmosregistration/common/routes.py

```diff
@@ -188,7 +191,7 @@ def __resource(self):
 
             else:
                 self.logger.writeInfo("POST resource response: {}".format(r.content))
-                abort(r.status_code)
+                return '', r.status_code
```

Comment:
We'd normally return a JSON body with the error to match the spec. I'd have thought abort would do this for us, but perhaps not. Was there some other reason for not using it?

Reply:
Well spotted, this is no longer needed.
Originally I was getting `LookupError: no exception for 201`, because of python 3 floating point division errors, which have been fixed. `if r.status_code // 100 == 2:`

Reply:
I note that flask has the following helper functions if we want to swap to using those?
is_informational()  # 1xx
is_success()        # 2xx
is_redirect()       # 3xx
is_client_error()   # 4xx
is_server_error()   # 5xx

Reply:
Probably best in the future, but we should check the Debian packaged version we depend upon is new enough to have them.